### ISSUE TITLE
broken siteconfig:  tvguide.com[Multi]

### ISSUE THREAD

**Issue by Toei79** [K-vanc/Tempest-EPG-Generator#164]
## Issue Type

Please choose your issue type

* [ ] Request
* [ ] Bug
* [x] Broken Siteconfig
* [ ] Question

## OS Details (Not required for Request)

Please choose your OS

* [ ] Windows
* [x] Unix
* [ ] Docker

Please write your version details (PHP version is not required for Docker)

* PHP Version: ...

* Tempest Version: ...rv2.0.0.0-24072013

Please write your web server name(Apache/Nginx etc.) for local servers, tick docker for docker image or tick shared host if you are running on a shared host service / server (VPS, Web Hosting etc.)
Ngnix
* Web Server Name: ...
* [ ] Docker
* [x] Shared Host

those are down,  
[ENC][EX]whattowatch.com[US]_0.channel
 
[ENC][EX]ontvtonight.com[US]_0.channel

edit, tvguide multi its fine i run again and worked? but when i click generate channels get me 1kb file but run again generate epg and worked fine.

**Comment by K-vanc** [comment]
Hi @Toei79;

* ontvtonight.com[US] grabs fine here but I will still update because noticed a problem on channel generation and director element. After update it will absolutely not work for you since you are using an outdated version of Tempest..

![image](https://github.com/user-attachments/assets/28054874-3c59-4fa2-ae6f-06ed058804b2)

* whattowatch website, there is no epg for couple of months. It can be an issue only if there is epg there but you cannot grab..

![image](https://github.com/user-attachments/assets/2072b6c1-50ec-4168-a3fc-c50b0fffa90c)

* tvguide works fine but you need to use zip code for channel creation.. It's explained here long time ago..

https://github.com/K-vanc/Tempest-EPG-Generator/blob/main/Siteconfigs/Multi%20Nation/Info%20for%20tvguide_com%20-%20i_mjh_nz%20-%20epgshare01.md

**Comment by Toei79** [comment]
About tvguide multi, I ran tempest and it wasn't loading them, that's why I thought that, and the channel line went to 1 kb, it usually worked fine but then I ran tempest again and it worked again, so maybe it's that there are too many requests made to the site I guess. I hadn't seen that you released a new version, I already updated it, thanks!

I think there are many sites down (in the US ones) but well there are many more working that generate the same lists basically. It's normal that they go down because of the changes they make.
